tests/behat/features/gridfield-navigation.feature

```diff
@@ -0,0 +1,161 @@
+Feature: Search in GridField
+  As an author
+  I want to search an item in the CMS
+  So that I see proper result and don't see warning
```

Comment:
This feature file is called "gridfield-navigation" - this information doesn't seem to be about navigation, it seems to be about search specifically.

Reply:
I suppose that "search" feature is not related to GridField navigation. So I extracted navigation steps from search feature and left it clean.

Reply:
Fix description.